pkg/config/defaults.go

```diff
@@ -0,0 +1,257 @@
+/*
+Copyright 2017 Kinvolk GmbH
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"os"
+	"os/exec"
+	"path"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/viper"
+
+	"github.com/kinvolk/kube-spawn/pkg/utils"
+	"github.com/kinvolk/kube-spawn/pkg/utils/fs"
+)
+
+const (
+	DefaultKubeSpawnDir      = "/var/lib/kube-spawn"
+	DefaultClusterName       = "default"
+	DefaultContainerRuntime  = RuntimeDocker
+	DefaultKubernetesVersion = "v1.7.5"
+	DefaultBaseImage         = "coreos"
+
+	DefaultDockerRuntimeEndpoint = "" // "unix:///var/run/docker.sock"
+	DefaultRktRuntimeEndpoint    = "unix:///var/run/rktlet.sock"
+	DefaultCrioRuntimeEndpoint   = "unix:///var/run/crio.sock"
+	DefaultRuntimeTimeout        = "15m"
+	DefaultRktStage1ImagePath    = "/usr/lib/rkt/stage1-images/stage1-coreos.aci"
+
+	CacheDir = ".cache"
```

Comment:
I don't think the directory should be hidden, as it leaves in `/var/lib/kube-spawn`.

Reply:
I wanted some kind of prefix as this makes distinguishing and [listing](https://github.com/kinvolk/kube-spawn/pull/161/commits/5b437dab9961f6c734701fd05c521aeecc0860a4#diff-04a5247871266df765044434b04e51acR58) cluster envs easier.

Reply:
The better way would be then to have an `environments` subfolder. Can be changed in the next version.